### PR TITLE
Add recipe for pastedeploy [skip appveyor] [skip travis]

### PR DESCRIPTION
**Pull request by lincrosenbach**

### CODE REVIEW

File: recipes/pastedeploy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - paste
+    - paste.deploy
+  requires:
```
Comment (nicoddemus) [inline recipes/pastedeploy/meta.yaml]: You can remove this `requires` section: we are not running the tests so it is not really needed.

File: recipes/pastedeploy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    - pytest
+    - pytest-runner
+
+about:
+  home: https://pylonsproject.org/
+  license: MIT
+  license_family: MIT
+  license_file: 'docs/license.txt'
```
Comment (igortg) [inline recipes/pastedeploy/meta.yaml]: fix that one

File: recipes/pastedeploy/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  summary: Load, configure, and compose WSGI applications and servers
+  doc_url: https://docs.pylonsproject.org/projects/pastedeploy/
```
Comment (igortg) [inline recipes/pastedeploy/meta.yaml]: This link seems broken

Comment (lincrosenbach) [reply recipes/pastedeploy/meta.yaml]: I dont think so.

File: recipes/pastedeploy/meta.yaml (hunk 2 above)
Comment (lincrosenbach) [reply recipes/pastedeploy/meta.yaml]: I need a hint...

File: recipes/pastedeploy/meta.yaml (hunk 1 above)
Comment (lincrosenbach) [reply recipes/pastedeploy/meta.yaml]: Removed

**Comment by lincrosenbach** [comment]
Miss setuptools for run deps. Added.

**Comment by lincrosenbach** [comment]
@conda-forge/staged-recipes @igortg I think this is ready for merge.

**Pull request merged**